Source/Core/Core/PowerPC/JitArm64/Jit.cpp

```diff
@@ -1308,6 +1271,7 @@ bool JitArm64::DoJit(u32 em_address, JitBlock* b, u32 nextPC)
 
   b->codeSize = static_cast<u32>(GetCodePtr() - b->normalEntry);
   b->originalSize = code_block.m_num_instructions;
+  b->cycle_count = js.downcountAmount;
```

Comment:
This is an oversimplification. The actual amount of cycles elapsed while running a block depends on how early we exit the block (if there's conditional branches in the block). That's why the old code writes out the value of js.downcountAmount at exit – because the value is different depending on what exit you take.

Reply:
Ah. That makes a lot of sense. I was looking for a way to make this information always available to my JitWidget refresh, but overlooked the fact that this information would vary. I will return it to the ProfileData struct and undo the oversimplification.